plugins/dbnd-docker/src/dbnd_docker/kubernetes/kubernetes_engine_config.py

```diff
@@ -92,15 +116,16 @@ class KubernetesEngineConfig(ContainerEngineConfig):
     requests = parameter.none()[Dict]
     limits = parameter.none()[Dict]
 
-    pod_exit_code_to_retry_count = parameter(empty_default=True).help(
-        "Mapping between pod exit code to amount of pod retry attempts"
+    pod_error_cfg_source_dict = parameter().help(
```

Comment:
according to description: pod_error_handling_config

Reply:
okok